src/components/Breadcrumb/Breadcrumb.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { v4 as uuid } from 'uuid';
+import { ThemeProvider } from 'styled-components';
+import {StyledBreadcrumb} from './Breadcrumb.styled'
+import themeSettings from './theme'
+import { BreadcrumbItem } from './BreadcrumbItem';
+
+const breadcrumbsList = [
+    {name: 'Home', href: '#', active: true},
+    {name: 'Library', href: 'https://getbootstrap.com/docs/4.0/components/breadcrumb/', active: true},
+    {name: 'Data', active: false}
```

Comment:
Lepiej byłoby dodać tutaj to `uuid()` jako kolejna właściwość

Reply:
Zmienione